app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new(post_id: params[:post_id], user_id: params[:user_id])
```

Comment:
- [ ] Hey there! 🌟 When creating a like, make sure to leverage the current_user established in your ApplicationController. 🤖 This ensures that the like is associated with the current logged-in user, maintaining accuracy and alignment with project requirements. Keep up the excellent work! 💪👩‍💻
```suggestion
 @post = Post.find(params[:post_id])
  @like = @current_user.likes.new(post_id: @post.id)
```